reference/catalog/brands_catalog.v3.yml

```
@@ -1908,6 +2449,76 @@ components:
       schema:
         type: string
         default: 'application/json'
+    PageParam:
+      name: page
+      description: |
+        Specifies the page number in a limited (paginated) list of products.
+      required: false
+      in: query
+      schema:
+        type: integer
+    MetafieldKeyParam:
+      name: key
+      in: query
+      description: Filter based on a metafieldʼs key.
+      required: false
+      schema:
+        type: string
+    MetafieldKeyInParam:
+      name: key:in
+      in: query
+      description: Filter based on comma separated metafieldʼs keys. Could be used with vanila 'key' query parameter.
+      required: false
+      style: form
+      explode: false
+      schema:
+        type: array
+        items:
+          type: string
+    MetafieldNamespaceParam:
+      name: namespace
+      in: query
+      description: Filter based on a metafieldʼs namespaces.
+      required: false
+      schema:
+        type: string
+    MetafieldNamespaceInParam:
+      name: namespace:in
+      in: query
+      description: Filter based on comma separated metafieldʼs namespaces. Could be used with vanila 'namespace' query parameter.
```

```suggestion
      description: Filter based on comma separated metafieldʼs namespaces. Could be used with vanilla `namespace` query parameter.
```
correct vanilla and swap straight quotes for backticks throughout this series of params.